tests/util/db_connection.py

```diff
@@ -2,16 +2,15 @@
 from chia.util.db_wrapper import DBWrapper2
 import tempfile
 import aiosqlite
-from datetime import datetime
-import sys
 
 
 async def log_conn(c: aiosqlite.Connection, name: str) -> aiosqlite.Connection:
-    def sql_trace_callback(req: str):
-        timestamp = datetime.now().strftime("%H:%M:%S.%f")
-        sys.stdout.write(timestamp + " " + name + " " + req + "\n")
-
     # uncomment this to debug sqlite interactions
+    # from datetime import datetime
+    # import sys
+    # def sql_trace_callback(req: str):
+    #    timestamp = datetime.now().strftime("%H:%M:%S.%f")
+    #    sys.stdout.write(timestamp + " " + name + " " + req + "\n")
```

Comment:
this is all code useful when debugging, so it might as well all be commented out

Reply:
I'd prefer to see it gated in a module-level global near the top, maybe "DO_SQL_LOGGING" rather than require uncommenting.

Reply:
Generally speaking the function doesn't need to be gated, it can just exist.  Even when it comes to coverage, we can make a separate job to enable this and make sure it keeps working.  Or...  `*shudder*`  a coverage ignore.  `:]`

For enabling it actually doing stuff, I would really like to avoid requiring code changes.  `pytest --chia-enable-sql-debugging` or such seems much better.

Reply:
*responsibility to create that can certainly be placed on me

Reply:
until very recently, the only way I knew of to set a breakpoint in pdb was to edit the source code to call `breakpoint()`. The `break` command in pdb doesn't work quite the same as gdb. So, I was under the impression that editing source code when debugging is common practice. But perhaps not.

Reply:
if leaving commented out code here is controversial, I'm happy to just delete it too. I don't think it's that valuable

Reply:
Editing code is certainly common for one-off diagnostic addition etc.  I think debugging code that is valuable enough to stick around should either just always be active in tests or configurable.  That encourages some formality and can help avoid pushing with the code modifications to enable the code in place.